src/server/types.rs

```diff
@@ -57,7 +57,7 @@ pub struct ModelSummary {
     pub name: String,
     pub version: String,
     pub description: Option<String>,
-    pub deployed: bool,
+    pub deployed_version: Option<String>,
```

Comment:
:alert: breaking change, this will need to be updated in `wash` as well

Reply:
I think this is more accurate and useful though! Do you?